src/site/content/en/blog/appcache-removal/index.md

```diff
@@ -0,0 +1,222 @@
+
+[Service workers](https://developers.google.com/web/fundamentals/primers/service-workers), which are widely supported in current browsers, offer an alternative to providing the offline experience that AppCache had offered.
+
+## Timeline
+
+[Recent changes](https://blog.chromium.org/2020/03/chrome-and-chrome-os-release-updates.html) to the Chrome release schedule means that the timing of some of these steps may vary. We will attempt to keep this timeline up-to-date, but at this point, please migrate off of AppCache as soon as possible, instead of waiting for specific milestones.
+
+A "deprecated" feature still exists, but logs warning messages discouraging use. A "removed" feature no longer exists in the browser.
+
+<div class="w-table-wrapper">
+  <table>
+    <tr>
+    <td><a href="https://groups.google.com/a/chromium.org/g/blink-dev/c/UKF8cK0EwMI/m/NLhsIrs-AQAJ">Deprecation in non-secure contexts</a>
+    </td>
+    <td>Chrome 50 (April 2016)
+    </td>
+    </tr>
+    <tr>
+    <td><a href="https://groups.google.com/a/chromium.org/g/blink-dev/c/ANnafFBhReY/m/1Xdr53KxBAAJ?pli=1">Removal from non-secure contexts</a>
+    </td>
+    <td>Chrome 70 (October 2018)
+    </td>
+    </tr>
+    <tr>
+    <td><a href="https://groups.google.com/a/chromium.org/g/blink-dev/c/FvM-qo7BfkI/m/0daqyD8kCQAJ">Deprecation in secure contexts</a>
+    </td>
+    <td>Chrome 79 (December 2019)
+    </td>
+    </tr>
+    <tr>
+    <td><a href="https://blog.chromium.org/2020/01/appcache-scope-restricted.html">AppCache scope restriction</a>
+    </td>
+    <td>Chrome 80 (February 2020)
+    </td>
+    </tr>
+    <tr>
+    <td>"Reverse" origin trial begins
+    </td>
+    <td>Chrome 84 (<a href="https://chromiumdash.appspot.com/schedule">estimated July 2020</a>)
+    </td>
+    </tr>
+    <tr>
+    <td><a href="https://groups.google.com/a/chromium.org/g/blink-dev/c/FvM-qo7BfkI/m/AvxoE6JpBgAJ">Removal from secure contexts</a>, except for those opted-in to the origin trial
+    </td>
+    <td>Chrome 85 (<a href="https://chromiumdash.appspot.com/schedule">estimated August 2020</a>)
+    </td>
+    </tr>
+    <tr>
+    <td>Complete removal from secure contexts for everyone, with completion of origin trial
+    </td>
+    <td>Chrome 90 (<a href="https://chromiumdash.appspot.com/schedule">estimated April 2021</a>)
+    </td>
+    </tr>
+  </table>
+</div>
+
+{% Aside %}
+This timeline applies to Chrome on **all platforms other than iOS**. There is also an adjusted timeline for AppCache used within an Android [WebView](https://developer.android.com/reference/android/webkit/WebView). For more info, see [The cross-platform story](#the-cross-platform-story) later in this post.
+{% endAside %}
```

Comment:
Would be nice to mention what happens to iOS here.

Reply:
There's a lot of nuance about iOS, so I have a link to `#the-cross-platform-story` and explain it there.